cmd/start.go

```diff
@@ -288,8 +287,22 @@ func runMasterNodOnHotHot(ctx context.Context, config *configs.Config) error {
 		return err
 	}
 
+	pastelPort, err := strconv.Atoi(pastelID)
```

Comment:
why Port is determined from pastelID???

Reply:
just follow current code. we need to verify again

Reply:
oh, i am just going over start.go logic now - this is total shit show